ci/do_ci.sh

```diff
@@ -227,6 +228,30 @@ if [[ "$CI_TARGET" == "bazel.release" ]]; then
   echo "bazel contrib release build..."
   bazel_contrib_binary_build release
 
+  exit 0
+elif [[ "$CI_TARGET" == "bazel.distribution" ]]; then
+  echo "Building distro packages..."
+
+  setup_clang_toolchain
+
+  # By default the packages will be signed by the first available key.
+  # If there is no key available, a throwaway key is created
+  # and the packages signed with it, for the purpose of testing only.
+  if ! gpg --list-secret-keys "*"; then
+      export PACKAGES_MAINTAINER_NAME="Envoy CI"
+      export PACKAGES_MAINTAINER_EMAIL="envoy@snake.oil"
```

Comment:
Is it real mail? I do not see it in the doc.

Reply:
it is not - and is intended to denote that - perhaps this could be more explicit in some way

Reply:
Use envoy@packaging.example.com perhaps?

Reply:
updated to envoy-ci@for.testing.only